invenio_theme/templates/semantic-ui/invenio_theme/page_cover.html

```diff
@@ -13,6 +13,11 @@
 
 {%- block body %}
   <div class="sixteen wide column centered">
+    {%- block site_banner %}
+      {% from 'invenio_banners/banner.html' import banner %}
```

Comment:
I don't think there is a direct dependency on invenio-banners in this module...I think it is better to either keep the block here and implement the banner in a "higher" level module or completely move this to another module where it depends on invenio-banners.